circus-train-aws/src/main/java/com/hotels/bdp/circustrain/aws/AssumeRoleCredentialProvider.java

```diff
@@ -29,7 +29,7 @@
 public class AssumeRoleCredentialProvider implements AWSCredentialsProvider {
 
   public static final String ASSUME_ROLE_PROPERTY_NAME = "com.hotels.bdp.circustrain.aws.AssumeRoleCredentialProvider.assumeRole";
-  private static final int CREDENTIALS_DURATION = 12 * 60 * 60; // max duration for assumed role credentials
+  private static final int CREDENTIALS_DURATION = 60 * 60; // max duration for assumed role credentials
```

Comment:
What unit is this measured in? Perhaps add this to the variable name or at the very least a comment stating this. I know this code predates your change but we might as well fix it.

Reply:
Why are we even changing this?

Reply:
Its in seconds, I've added it to the comment. Think Abhi changed this when it was timing out and we were trying to figure out why. Will change it back

Reply:
Found out why we changed it: the role we're using for testing has a max duration of 1 hour, but this duration was set to 12 hours so it was failing. Will need to increase the duration of the role.